tests/server/api/test_deployments.py

```diff
@@ -1390,6 +1390,84 @@ async def test_get_scheduled_runs_sort_order(
             str(flow_run.id) for flow_run in flow_runs[:3]
         ]
 
+    async def test_get_scheduled_flow_runs_updates_last_polled_time_and_status(
```

Comment:
Can we add a check for a `READY` status in these tests?

Reply:
Ah yeah for sure, I knew something was missing last night

Reply:
Applied in [23130a0](https://github.com/PrefectHQ/prefect/pull/10969/commits/23130a0c575ca9eaa37beab0c472e182615331ef)